tools/docker/Dockerfile.monorepo

```diff
@@ -46,8 +46,8 @@ RUN --mount=type=cache,target=/var/lib/apt/lists/,sharing=private \
         "php${PHP_VERSION}-xml" \
         "php${PHP_VERSION}-zip" \
         rsync \
-    && apt-get remove --purge --auto-remove -y gpg software-properties-common \
-    && find /var/ -name '*-old' -delete && rm -rf /var/log/dpkg.log /var/log/alternatives.log /var/log/apt/ ~/.launchpadlib
```

Comment:
Unless they've changed it recently, each `RUN` builds a "layer" that modifies the previous layer. Removing this `find` command means this layer will still contain these files.

I see some of the later `RUN`s doing an `apt-get install` below also lost the `find` at some point.